### PR TITLE
Kaa-876: Device management

### PR DESCRIPTION
**Pull request by ashvayka**

### CODE REVIEW

**Comment by kaa-leeroy** [comment]
Can one of the admins verify this patch?

**Comment by rasendubi** [comment]
add to whitelist

**Comment by rasendubi** [comment]
I see whole feature/KAA-876 branch as well as new commits that weren't reviewed previously. Could you please split the PR into two parts and pass code review for new changes according to process?

**Comment by sashadidukh** [comment]
LGTM

**Comment by ashvayka** [comment]
@rasendubi How exactly should I split this PRs? What items do you want to review and why this can't be done in this PR?

**Comment by ashvayka** [comment]
@ikulikov please review

File: client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/channel/failover/DefaultFailoverManager.java [hunk 1]
```diff
@@ -16,21 +16,20 @@
```
Comment (forGGe) [inline client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/channel/failover/DefaultFailoverManager.java:16]: Commit message typo: `imrpvements` should be `improvements`

**Comment by ikulikov** [comment]
LGTM

**Pull request merged**